_docs/testing.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Testing
+---
+
+# Testing
```

Comment:
Please remove it. It's duplicate otherwise because h1 is injected from `title` front matter attribute

Reply:
I was also wondering about this but haven't been able to quickly find the answer or examplesonline.

I'll remove it